### PR TITLE
completed

### PR DESCRIPTION
**Pull request by OleksandrProkop**
[DEMO LINK](https://OleksandrProkop.github.io/react_tabs-with-router/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Navigate, NavLink, Route, Routes } from 'react-router-dom';
+import classNames from 'classnames';
+import { TabPage } from './components/TabsPage';
```
Comment (max-rozzhalovets) [inline src/App.tsx:6]:
```suggestion
import { TabsPage } from './components/TabsPage';
```
It's better to use the same name with the file

File: src/App.tsx [hunk 2]
```diff
@@ -17,35 +20,57 @@ export const App = () => (
...
+          </NavLink>
+          <NavLink
+            to="/tabs"
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'is-active': isActive },
+            )}
+          >
+            Tabs
+          </NavLink>
```
Comment (max-rozzhalovets) [inline src/App.tsx:40]: Create files for separate components like this one

File: src/components/TabsPage/TabsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {selectTab
+          ? selectTab.content
+          : 'Please select a tab'}
+      </div>
```
Comment (max-rozzhalovets) [inline src/components/TabsPage/TabsPage.tsx:39]: Suggest to create file for this part also

File: src/components/TabsPage/TabsPage.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              data-cy="Tab"
+              className={classNames({ 'is-active': tab.id === tabId })}
+              key={tab.id}
+            >
+              <Link
+                to={`/tabs/${tab.id}`}
```
Comment (max-rozzhalovets) [inline src/components/TabsPage/TabsPage.tsx:25]: You can make a relative path

**Review by max-rozzhalovets — APPROVED** [review]
There are some inaccuracies, but I guess they are not critical. Mostly good job

File: src/App.tsx (hunk 1 above)
Comment (Anna-Tkachenko) [reply src/App.tsx:6]: check @max-rozzhalovets comment

File: src/App.tsx (hunk 2 above)
Comment (Anna-Tkachenko) [reply src/App.tsx:40]: check @max-rozzhalovets comment

File: src/App.tsx [hunk 5]
```diff
@@ -17,35 +20,57 @@ export const App = () => (
...
 
     <div className="section">
       <div className="container">
-        <h1 className="title">Home page</h1>
-        <h1 className="title">Tabs page</h1>
-        <h1 className="title">Page not found</h1>
+        <Routes>
+          <Route
+            path="*"
+            element={<h1 className="title">Page not found</h1>}
```
Comment (Anna-Tkachenko) [inline src/App.tsx:50]: let's move elements to the separate components

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — APPROVED** [review]
Good job! ⭐️

**Pull request closed without merging**